plaidml/bridge/openvino/tests/functional/plugin/plaidml/shared_tests_instances/single_layer_tests/reshape.cpp

```diff
@@ -15,15 +15,15 @@ const std::vector<InferenceEngine::Precision> netPrecisions = {
     // InferenceEngine::Precision::I64
 };
 
-INSTANTIATE_TEST_SUITE_P(ReshapeCheckDynBatch, ReshapeLayerTest,
-                         ::testing::Combine(::testing::Values(true),                                      //
-                                            ::testing::ValuesIn(netPrecisions),                           //
-                                            ::testing::Values(std::vector<size_t>({1, 16, 16, 16})),      //
-                                            ::testing::Values(std::vector<size_t>({1, 0, 256})),          //
-                                            ::testing::Values(CommonTestUtils::DEVICE_PLAIDML),           //
-                                            ::testing::Values(std::map<std::string, std::string>(         //
-                                                {{CONFIG_KEY(DYN_BATCH_ENABLED), CONFIG_VALUE(YES)}}))),  //
-                         ReshapeLayerTest::getTestCaseName);
+// INSTANTIATE_TEST_SUITE_P(ReshapeCheckDynBatch, ReshapeLayerTest,
+//                          ::testing::Combine(::testing::Values(true),                                      //
+//                                             ::testing::ValuesIn(netPrecisions),                           //
+//                                             ::testing::Values(std::vector<size_t>({1, 16, 16, 16})),      //
+//                                             ::testing::Values(std::vector<size_t>({1, 0, 256})),          //
+//                                             ::testing::Values(CommonTestUtils::DEVICE_PLAIDML),           //
+//                                             ::testing::Values(std::map<std::string, std::string>(         //
+//                                                 {{CONFIG_KEY(DYN_BATCH_ENABLED), CONFIG_VALUE(YES)}}))),  //
+//                          ReshapeLayerTest::getTestCaseName);
```

Comment:
@tzerrell @cnamrata15 @brianretford These tests now appear to crash with the latest changes. I wasn't able to determine exactly why, but it seems similar to the full network issues that were reported recently. Is it OK to drop these for now?